src/gui/vsDevice.cpp

```diff
@@ -296,6 +315,25 @@ void VsDevice::reconcileAgentConfig(QJsonDocument newState)
     }
 }
 
+// initPinger intializes the pinger used to generate network latency statistics for
+// Virtual Studio
+VsPinger* VsDevice::startPinger(VsServerInfo* studioInfo)
+{
+    QString id        = studioInfo->id();
+    QString sessionId = studioInfo->sessionId();
+    QString host      = studioInfo->sessionId();
+    host.append(QString::fromStdString(".jacktrip.cloud"));
+
+    m_pinger = new VsPinger(QString::fromStdString("wss"), host,
+                            QString::fromStdString("/ping"));
+}
+
+// stopPinger stops the Virtual Studio pinger
+void VsDevice::stopPinger()
```

Comment:
Where is `stopPinger()` called? Ideally we only setup this pinger when connected to a studio and teardown once disconnected

Reply:
added `stopPinger` right before `stopJackTrip` is called - left it out by mistake.